simul/platform/cliutils.go

```diff
@@ -33,16 +36,24 @@ func Scp(username, host, file, dest string) error {
 // Rsync copies files or directories to the remote host. If the DebugVisible
 // is > 1, the rsync-operation is displayed on screen.
 func Rsync(username, host, file, dest string) error {
-	addr := host + ":" + dest
+	h, p, err := net.SplitHostPort(host)
+	if err != nil && strings.Contains(err.Error(), "missing port in address") {
+		p = "22"
+		err = nil
+	}
+	if err != nil {
+		return err
+	}
+	addr := h + ":" + dest
 	if username != "" {
 		addr = username + "@" + addr
 	}
-	cmd := exec.Command("rsync", "-Pauz", "-e", "ssh -T -o Compression=no -x", file, addr)
+	cmd := exec.Command("rsync", "-Pauz", "-e", fmt.Sprintf("ssh -T -o Compression=no -x -p %v", p), file, addr)
```

Comment:
I prefer `%d` over `%v` to indicate which format I want.

```suggestion
	cmd := exec.Command("rsync", "-Pauz", "-e", fmt.Sprintf("ssh -T -o Compression=no -x -p %d", p), file, addr)
```